Split.xcodeproj/xcshareddata/xcschemes/Split.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
ℹ️ Generated by `swift package generate-xcodeproj` as `Split-Package.xcscheme`, renamed to `Split.xcscheme`